src/control/cmd/daos/acl.go

```diff
@@ -0,0 +1,324 @@
+//
+// (C) Copyright 2021 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package main
+
```

Comment:
Maybe a comment here to say that this file is for parsons & converting ACL input strings into DAOS ACL data structure.